great_expectations/expectations/core/expect_column_mean_to_be_between.py

```diff
@@ -144,8 +145,9 @@ class ExpectColumnMeanToBeBetween(ColumnExpectation):
                     "estimator": "bootstrap",
                     "n_resamples": 9999,
                     "random_seed": None,
-                    "quantile_statistic_interpolation_method": "auto",
-                    "quantile_bias_std_error_ratio_threshold": 0.25,
+                    "quantile_statistic_interpolation_method": "nearest",
```

Comment:
@NathanFarmer Let us leave `quantile_statistic_interpolation_method` as "auto" -- thanks.

Reply:
This will remain `nearest` per our offline conversation.